docs/making-release-notes.md

```diff
@@ -0,0 +1,38 @@
+## Making release notes
+This documents the process for making release notes for a release.
+
+### 1) Note the PR number of the previous release
+Find the PR that was merged with the previous release.  Remember this number
```

Comment:
It seems like there should be a simple command line that would generate this that we can document here.